services/obs/obs.tester.js

```diff
@@ -0,0 +1,25 @@
+import { ServiceTester } from '../tester.js'
+import { noToken } from '../test-helpers.js'
+import ObsService from './obs.service.js'
+import { isBuildStatus } from './obs-build-status.js'
+
+export const t = new ServiceTester({
+  id: 'obs',
+  title: 'openSUSE Open Build Service',
+})
+
+t.create('status (valid)')
+  .skipWhen(noToken(ObsService))
+  .get('/openSUSE:Factory/aaa_base/standard/x86_64.json?label=standard')
+  .expectBadge({
+    label: 'standard',
+    message: isBuildStatus,
+  })
+
+t.create('status (invalid)')
+  .skipWhen(noToken(ObsService))
+  .get('/home:sp1rit/this_package_will_never_exist/repo/arch.json')
+  .expectBadge({
+    label: 'OBS',
+    message: 'Package not found',
+  })
```

Comment:
I've wired up the CI and review app environments with the relevant auth configuration (https://shields-staging-pr-6993.herokuapp.com/obs/openSUSE:Tools/osc/Debian_11/x86_64) so the tests should now be getting executed. Think these will need to be updated in order to pass, to account for some of the label and message changes to the services

Reply:
ah, I forgot to update the invalid test 🤦🏻‍♂️ 

should be fixed now